internal/donor/donordata/provided.go

```diff
@@ -175,9 +175,14 @@ type Provided struct {
 	HasSentApplicationUpdatedEvent bool `hash:"-"`
 }
 
-// CanChange returns true if the donor can make changes to their LPA.
-func (p *Provided) CanChange() bool {
-	return p.SignedAt.IsZero()
+// CanChangePersonalDetails returns true if the donor can make changes to their FirstNames, LastName or DateOfBirth.
```

Comment:
I'd prefer us to keep `CanChange` too then, and have `CanChangePersonalDetails` use that instead of `!p.SignedAt.IsZero`. I just don't trust that the condition won't change in future, and it is a bit nicer to read

Reply:
Fair point - I was on the fence as it was only checking one prop but, like you said, this could change.